shared/templates/sudo_defaults_option/bash.template

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+if /usr/sbin/visudo -qcf /etc/sudoers; then
+    if ! grep -P "^(?!#)[\s]*Defaults.*[\s]+{{{ OPTION }}}[\s]*.*$" /etc/sudoers; then
+        cp /etc/sudoers /etc/sudoers.bak
```

Comment:
I would probably consider backup in `/tmp`, but that's just a suggestion.

Reply:
I don't have a strong opinion on this.
I followed approach similar to:
https://github.com/ComplianceAsCode/content/blob/master/shared/macros-bash.jinja#L465